src/main/java/com/trailfinder/TrailFinderController.java

```diff
@@ -23,7 +23,6 @@
 @Controller
 public class TrailFinderController {
 	
-	// Private Fields
```

Comment:
Removed unnecessary comment

Reply:
Agreed.  You can determine that by looking at the access modifier.